src/third_party/eris/loslib.c

```diff
@@ -102,14 +102,11 @@ static time_t l_checktime (lua_State *L, int arg) {
 /*
 ** {==================================================================
 ** Configuration for 'tmpnam':
-** By default, Lua uses tmpnam except when POSIX is available, where
-** it uses mkstemp.
+** By default, Lua uses POSIX mkstemp.
 ** ===================================================================
 */
 #if !defined(lua_tmpnam)	/* { */
 
-#if defined(LUA_USE_POSIX)	/* { */
```

Comment:
Please document this change in `third_party/README`. We will need to reapply it every time we want to upgrade to a new Eris version.